lib/iris/fileformats/pp.py

```diff
@@ -673,8 +673,8 @@ def __init__(self, value):
 
         """
         if value < 0:
-            raise ValueError('Negative numbers not supported with '
-                             'splittable integers object')
+            msg = 'Setting LBPROC with a negative value of {}'
```

Comment:
Also if a negative LBPROC caused a downstream failure having a warning to indicate something non-standard is going on could prove useful for troubleshooting the eventual failure...